__tests__/__unit__/shared/utils.unit.test.ts

```diff
@@ -12,17 +12,183 @@
 import * as utils from "../../../src/utils";
 import * as sharedUtils from "../../../src/shared/utils";
 import * as globals from "../../../src/globals";
+import { Session, IProfileLoaded, Logger } from "@zowe/imperative";
 import { ZoweDatasetNode } from "../../../src/dataset/ZoweDatasetNode";
 import * as vscode from "vscode";
 import * as path from "path";
-import {
-    generateIProfile,
-    generateISessionWithoutCredentials
-} from "../../../__mocks__/mockCreators/shared";
-import { generateDatasetSessionNode } from "../../../__mocks__/mockCreators/datasets";
+import { createIProfile, createISessionWithoutCredentials, createISession, createFileResponse } from "../../../__mocks__/mockCreators/shared";
+import { createDatasetSessionNode } from "../../../__mocks__/mockCreators/datasets";
+import { Profiles } from "../../../src/Profiles";
+import { ZoweUSSNode } from "../../../src/uss/ZoweUSSNode";
+import { Job } from "../../../src/job/ZoweJobNode";
+import { ZoweExplorerApiRegister } from "../../../src/api/ZoweExplorerApiRegister";
 
 jest.mock("path");
 
+async function createGlobalMocks() {
+    const newVariables = {
+        session: createISession(),
+        profileOne: createIProfile(),
+        mockLoadNamedProfile: jest.fn()
+    };
+
+    Profiles.createInstance(Logger.getAppLogger());
+    newVariables.mockLoadNamedProfile.mockReturnValue(newVariables.profileOne);
+    Object.defineProperty(Profiles, "getInstance", {
+        value: jest.fn(() => {
+            return {
+                allProfiles: [{name: "firstName"}, {name: "secondName"}],
+                getDefaultProfile: {name: "firstName"},
+                loadNamedProfile: newVariables.mockLoadNamedProfile
+            };
+        }),
+        configurable: true
+    });
+
+    return newVariables;
+}
+
+describe("Shared Utils Unit Tests - Function node.labelRefresh()", () => {
+    it("Checks that labelRefresh subtly alters the label", async () => {
+        const globalMocks = await createGlobalMocks();
+        const rootNode = new ZoweUSSNode(
+            "gappy", vscode.TreeItemCollapsibleState.Collapsed, null, globalMocks.session, null, false, null, undefined);
+        expect(rootNode.label === "gappy");
+        sharedUtils.labelRefresh(rootNode);
+        expect(rootNode.label === "gappy ");
+        sharedUtils.labelRefresh(rootNode);
+        expect(rootNode.label === "gappy");
+    });
+});
+
+describe("Positive testing", () => {
+    it("should pass for ZoweDatasetTreeNode with ZoweDatasetNode node type", async () => {
+        const dsNode = new ZoweDatasetNode(null, null, null, null);
+        const value = sharedUtils.isZoweDatasetTreeNode(dsNode);
+        expect(value).toBeTruthy();
```

Comment:
Note: I see the use of `.toBeTruthy` a lot. I think it is better to use explicit values when possible like`.toBe(true)`. Not sure if it applies here. No action requied.